build/bin/build-server-rpm

```diff
@@ -238,6 +238,19 @@ d }\" \
 
                 spectool -C SOURCES -g SOURCES/${spec_name}.spec
 
+                rpmbuild --define '_rpmdir %{_topdir}/RPMS/${spec_name}-${rpm_version}' --define 'dist .el7' -ba SOURCES/${spec_name}.spec
+                rm -f SOURCES/${spec_name}.spec*
+	    elif [ \"${spec_name}\" == \"percona-qan-app\" ]; then
+	        sudo yum remove -y nodejs npm
```

Comment:
There seems to be some whitespace problem.

Reply:
fixed